retrofit/src/main/resources/META-INF/proguard/rules.pro

```diff
@@ -0,0 +1,10 @@
+# Retain generic type information for use by reflection by converters and adapters.
```

Comment:
The file names are irrelevant so we can name them whatever we want. There's an issue to automatically strip them from the APK. We can use the module name for them to prevent collision though.